src/ptvsd/_vendored/pydevd/_pydev_bundle/pydev_monkey.py

```diff
@@ -61,6 +62,19 @@ def _is_managed_arg(arg):
 
 
 def _on_forked_process(setup_tracing=True):
+    pydevd_constants.after_fork()
+    pydevd_constants.DebugInfoHolder.DEBUG_STREAM = NULL
+
+    # TODO: We should use the pid for logging in pydevd to deal with multiple processes.
```

Comment:
Note: I created https://github.com/microsoft/ptvsd/issues/1831 to address this properly.